config/initializers/unused_identity_config_keys.rb

```diff
@@ -0,0 +1,3 @@
+if !IdentityConfig.unused_keys.empty?
```

Comment:
Personal style, I prefer positive checks over negative ones where it's easy
```suggestion
if IdentityConfig.unused_keys.present?
```